.circleci/config.yml

```diff
@@ -43,7 +44,7 @@ jobs:
             - npm-cache-{{ checksum "package-lock.json" }}
       - run:
           name: Install node dependencies
-          command: npm install
+          command: npm ci
```

Comment:
Switched to `npm ci` (https://docs.npmjs.com/cli/v10/commands/npm-ci) so installing doesn't affect the package lock and we can take advantage of caching